ert_shared/services/_base_service.py

```diff
@@ -206,10 +202,10 @@ def _do_shutdown(self) -> None:
             return
         try:
             self._proc.terminate()
-            self._proc.wait(10)
+            self._proc.wait(10)  # Give it 10s to shut down first..
         except TimeoutExpired:
-            self._proc.kill()
-            self._proc.wait()
+            self._proc.kill()  # ... then kick it harder...
+            self._proc.wait(self._timeout)  # ... and wait a little longer
```

Comment:
So the way this was setup before was that it would simply hang on `wait` if the process was not killed?  And now we could end up returning without process being completely killed? Should we then check the return value of `wait` (which will be false in the given scenario of timeout reached)? (not sure what else to do at this stage though.... Perhaps at least some logging would be beneficial)

Reply:
My understanding of existing code is that it waits 10s after trying terminate(), then tries kill() and waits forever. I don't think this is a big problem, but it feels a little safer to allow a timeout also in the second wait, and I'll definitely add logging